svc-bie-kafka/src/test/java/gov/va/vro/services/bie/config/KafkaConsumerCreatorTest.java

```diff
@@ -1,53 +1,24 @@
 package gov.va.vro.services.bie.config;
 
-import static org.assertj.core.api.Assertions.assertThat;
-
 import gov.va.vro.services.bie.service.AmqpMessageSender;
-import gov.va.vro.services.bie.service.kafka.KafkaConsumerCreator;
+import gov.va.vro.services.bie.service.kafka.KafkaConsumer;
 import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
 import org.springframework.kafka.core.ConsumerFactory;
 
-import java.util.Map;
-
 @ExtendWith(MockitoExtension.class)
 class KafkaConsumerCreatorTest {
 
   @Mock private ConsumerFactory<?, ?> consumerFactory;
   @Mock private AmqpMessageSender amqpMessageSender;
 
   private BieProperties bieProperties;
-  private KafkaConsumerCreator kafkaConsumerCreator;
+  private KafkaConsumer kafkaConsumer;
```

Comment:
souldn't you change the file/class name to too?

Reply:
Addressed